bootstrap.js

```diff
@@ -5,6 +5,12 @@ module.exports = function(grunt) {
     config: config
   });
 
+  // Allow Gruntconfig.json to set default descriptions.
+  if (config.help) {
+    grunt.config('help', config.help);
+    grunt.config('config.help', '');
```

Comment:
So Gruntconfig will get a 'help' key. When we load it into the grunt.config, it's located at grunt.config('config.help'). This code snippet is intended to move grunt.config('config.help') to grunt.config('help'). As an alternative, we could set up the help task to process config.help, but that seems super verbose.